src/wp-includes/default-filters.php

```diff
@@ -752,4 +752,37 @@
 add_action( 'before_delete_post', '_wp_before_delete_font_face', 10, 2 );
 add_action( 'init', '_wp_register_default_font_collections' );
 
+// TODO: This should probably go somewhere else.
```

Comment:
Perhaps this function might be best placed in `block-template-utils.php`

Reply:
Ah, good call!

As an alternative, maybe it's worth putting directly into `WP_REST_Templates_Controller` after all? I was hesitant to have it directly inside a controller when I thought that templates were handled by `WP_REST_Posts_Controller` (as it seemed too specific to the `wp_template` and `wp_template_part` post types there), but realizing that we have a specialized controller, it might not feel so out of place there... 🤔